internal/cli/test.go

```diff
@@ -360,3 +352,48 @@ func (c *cli) customDomainPickerOptions() (pickerOptions, error) {
 
 	return opts, nil
 }
+
+func (c *cli) appPickerWithCreateOption() (pickerOptions, error) {
+	options, err := c.appPickerOptions()
+	if err != nil {
+		return nil, err
+	}
+
+	enhancedOptions := []pickerOption{
+		{
+			value: newClientOption,
```

Comment:
Maybe the value could be a known constant like e.g. `NEW_CLIENT` specifically to be used as a key?

Reply:
🤔 Isn't it already a constant? It's just not called `NEW_CLIENT` but `newClientOption`, or are you suggesting to rename it?

Reply:
I mean to use different constants for value and label. Now it's using the label for both.

Reply:
Ah, understood now. Yeah it makes sense 👍🏻 Updated in [`eae18c8` (#629)](https://github.com/auth0/auth0-cli/pull/629/commits/eae18c80c91f4a77be0ca7220dc4b4dc663744b3)